src/pages/home/components/product-filters/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box } from "@mui/material";
+import React, { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { useGetFiltersQuery } from "~/redux/filters/filters-api";
+import { setFilters } from "~/redux/filters/filters.slice";
+import { useAppDispatch, useAppSelector } from "~/redux/hooks";
+
+type ProductFiltersProperties = {};
```

Comment:
Can we delete type here, if we don't use Properties for now?

Reply:
good question

Reply:
Yes, I just left it for further work on this task, but okay I'll delete it